fineract-provider/src/main/resources/sql/migrations/core_db/V310__multiselect_for_reportparameters.sql

```diff
@@ -0,0 +1,10 @@
+UPDATE `report`.`stretchy_parameter` SET `parameter_sql`='(select lo.id, lo.display_name as `Name` 
```

Comment:
Why this script required?

Reply:
@jinjurajan : Can you explain this query and share a link to the JIRA ticket so that we can pull this in?